### ISSUE TITLE
Add "Last Updated" section to each page

### ISSUE THREAD

**Issue by coriobriensfdo** [SFDO-Community-Sprints/DLRS-Documentation#40]
If we add a "Last Updated" line with date and person to each page we can keep track of pages updates/accuracy going forward.

**Comment by mkolodner** [comment]
@tcdahlberg - Is there, by any chance, some kind of tool that can be installed into GitHub that would do this for us? (Based on the last commit for the page, add a block at the bottom listing when it was updated and by whom.)

**Comment by mkolodner** [comment]
Yes, please, @tcdahlberg ! (Unless anyone else objects? @sharicarlson13 @coriobriensfdo ?)
Personally I would not include the comments tool, just the Last Modified info. But others might feel differently on that.

**Comment by sharicarlson13** [comment]
@mkolodner @tcdahlberg Yes, please! 

I'm okay not including the comments tool. Though we may want to consider in the future as the feedback piece.

**Comment by coriobriensfdo** [comment]
WOW this is awesome!! Thank you Thad! 

I guess my question is, if is the downside to collecting feedback on the page? I could see folks using that to ask questions when we'd want to point them to the TBC group instead I guess....

**Comment by mkolodner** [comment]
I just tend to think all "comments" sections are nothing but an invitation to spam and trolls. Plus unless one of us is going to be monitoring all those comments, I think it could just be a black hole. At least TBC is a place we're watching.

**Comment by coriobriensfdo** [comment]
Agree and makes sense! Let's leave off for now and strategize our feedback plan as you suggest.

**Comment by tcdahlberg** [comment]
I wasn't actually showing off the comments section, though I can add that if you want. Commenting requires a github account to comment which cuts down on trolling. Comments become issues in you repository where you can deal with them (delete, etc). The idea is that you aren't creating a third party database somewhere to deal with comments and just using standard GitHub tech. At any rate, the last updated is very separate code from commenting.

**Comment by coriobriensfdo** [comment]
> I wasn't actually showing off the comments section, though I can add that if you want. Commenting requires a github account to comment which cuts down on trolling. Comments become issues in your repository where you can deal with them (delete, etc). The idea is that you aren't creating a third party database somewhere to deal with comments and just using standard GitHub tech. At any rate, the last updated is very separate code from commenting.

Hmmmmmmmmm it would be pretty fantastic to have that one day...
